src/sinks/loki/config.rs

```diff
@@ -53,16 +53,26 @@ impl Default for CompressionConfigAdapter {
     }
 }
 
+fn default_loki_path() -> String {
+    "/loki/api/v1/push".to_string()
+}
+
 /// Configuration for the `loki` sink.
 #[configurable_component(sink("loki"))]
 #[derive(Clone, Debug)]
 #[serde(deny_unknown_fields)]
 pub struct LokiConfig {
     /// The base URL of the Loki instance.
     ///
-    /// Vector will append `/loki/api/v1/push` to this.
+    /// Vector will append the value of `path` to this.
     pub endpoint: UriSerde,
 
+    /// The path to use in the URL of the Loki instance.
+    ///
+    /// Default to `/loki/api/v1/push`.
```

Comment:
```suggestion
    /// By default, `"/loki/api/v1/push"` is used.
```

This appears to be the style we've used for other notes about default values.